Google.Ads.GoogleAds/examples/CampaignManagement/ValidateAd.cs

```diff
@@ -20,21 +20,21 @@
 using Google.Ads.GoogleAds.V11.Resources;
 using Google.Ads.GoogleAds.V11.Services;
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using static Google.Ads.GoogleAds.V11.Enums.AdGroupAdStatusEnum.Types;
+using static Google.Ads.GoogleAds.V11.Enums.ServedAssetFieldTypeEnum.Types;
 using static Google.Ads.GoogleAds.V11.Errors.PolicyFindingErrorEnum.Types;
 
 namespace Google.Ads.GoogleAds.Examples.V11
 {
     /// <summary>
-    /// This code example shows how to use the validateOnly header to validate an expanded text ad.
+    /// This code example shows how to use the validateOnly header to validate a responsive search ad.
```

Comment:
Says "header" but you may have been thinking of the AdWords API where that was a header.
Could we say "validateOnly request parameter" instead?

Reply:
You're right, addressing this in a separate PR.

Reply:
#446